### PR TITLE
Fix action and add relation test

### PR DESCRIPTION
**Pull request by jsangmeister**
see https://github.com/OpenSlides/openslides-backend-coordination/issues/64

@normanjaeckel can you take a look at this? I added a test for it and it seems like structured relations are not updated automatically.

### CODE REVIEW

**Comment by jsangmeister** [comment]
Rebased on #328. Test was not fixed, since it's another case (reverse structured field of a structured field). So please fix this @normanjaeckel

**Comment by normanjaeckel** [comment]
Please rebase again on #328. I added some fixes to #328 that might help also for this test here.

**Comment by jsangmeister** [comment]
Rebased. Test is not fixed.

**Comment by normanjaeckel** [comment]
Fixed. Please reflect change also in #337.

File: openslides_backend/action/relations.py [hunk 1]
```diff
@@ -227,7 +227,7 @@ def perform(self) -> Relations:
         #  - All other fields.
         if self.field.generic_relation:
             return self.prepare_result_to_fqid(add, remove, rels, related_name)
-        elif self.field.structured_relation:
+        elif self.field.structured_relation or self.field.structured_tag:
```
Comment (normanjaeckel) [inline openslides_backend/action/relations.py:230]: This was the fix.

Comment (jsangmeister) [reply openslides_backend/action/relations.py:230]: Well, that was easy...

**Comment by jsangmeister** [comment]
> Fixed. Please reflect change also in #337.

Yes, thank you. Will merge this probably after #337 and take care of the merge conflicts here.

**Pull request merged**